MediaRecorder.html

```diff
@@ -92,22 +92,30 @@
     <dd>The value of the Audio encoding target bit rate that was passed to the Platform (potentially truncated, rounded, etc), or the calculated one if the user has specified <code>bitsPerSecond</code>.</dd>
 
     <dt>void start()</dt>
-    <dd>When a <code>MediaRecorder</code> object’s <code>start()</code> method is invoked, the UA <em title="must" class="rfc2119">must</em> queue a task, using the DOM manipulation task source, that runs the following steps:
-     <ol>
-      <li>If the <code>state</code> is not "inactive", throw a <code>InvalidStateError</code> DOMException and terminate these steps. Otherwise:</li>
-      <li>Set <code>state</code> to 'recording' and wait until media becomes available from <code>stream</code>.</li>
-      <li>Once data becomes available fire a <code>start</code> event and start gathering the data into a Blob (see [[!FILE-API]]). </li>
-      <li>If the <code>timeSlice</code> argument has been provided, then once <code>timeSlice</code> milliseconds of data have been colleced, or some minimum time slice imposed by the UA, whichever is greater, fire a <code>dataavailable</code> event containing the Blob of collected data, and start gathering a new Blob of data.  Otherwise (if <code>timeSlice</code> has not been provided), continue gathering data into the original Blob.</li>
-      <li>When the <code>stream</code> is ended set <code>recording</code> to 'inactive' and stop gathering data. Callers <em title="should" class="rfc2119">should</em> not rely on exactness of the timeSlice value, especially if the timeSlice value is small. Callers <em title="should" class="rfc2119">should</em> consider timeSlice as a minimum value</li>
-      <li>Then fire a <code>dataavailable</code> event containing the Blob of data.</li>
-      <li>Finally, fire a <code>stop</code> event.</li>
-     </ol>
+    <dd>When a <code>MediaRecorder</code> object’s <code>start()</code> method is invoked, the UA <em title="must" class="rfc2119">must</em> run the following steps:
+      <ol>
+        <li>Let <var>timeslice</var> be the method's first argument, if provided, or <code>undefined</code>.</li>
+        <li>If <code>state</code> is not "inactive", throw an <code>InvalidStateError</code> DOMException and abort these steps.</li>
+        <li>If the stream's isolation properties disallow access from this MediaRecorder, throw a <code>SecurityError</code> DOMException and abort these steps.</li>
+        <li>Set <code>state</code> to "recording", and run the following steps in parallel:</li>
```

Comment:
@miguelao Fyi it is defined [here](https://html.spec.whatwg.org/#in-parallel). How do you want to handle it? FWIW the webrtc spec uses "in parallel" without defining it.